### ISSUE TITLE
Sending in malformed page token causes 500 

### ISSUE THREAD

**Issue by david4096** [ga4gh/server#982]
I was testing out the pageToken interface and found that I could reliably cause the server to 500 by sending in pagetokens like; `1:1`.

```
127.0.0.1 - - [14/Mar/2016 09:25:54] "POST /reads/search HTTP/1.1" 500 -
{u'bioSampleId': u'YnJjYTE6SEcwMDA5Ne', u'pageToken': u'1:1', u'referenceId': u'aGczNzpyZWZfYnJjYTE', u'readGroupIds': [u'YnJjYTE6SEcwMDA5NjpTUlIwNjI2MzQ']}
--------------------------------------------------------------------------------
ERROR in app [/Users/david/server-1/server-1/lib/python2.7/site-packages/flask/app.py:1423]:
Exception on /reads/search [POST]
--------------------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/david/server-1/server-1/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/david/server-1/server-1/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/david/server-1/ga4gh/frontend.py", line 485, in searchReads
    flask.request, app.backend.runSearchReads)
  File "/Users/david/server-1/ga4gh/frontend.py", line 409, in handleFlaskPostRequest
    return handleHttpPost(flaskRequest, endpoint)
  File "/Users/david/server-1/ga4gh/frontend.py", line 285, in handleHttpPost
    responseStr = endpoint(request.get_data())
  File "/Users/david/server-1/ga4gh/backend.py", line 773, in runSearchReads
    self.readsGenerator)
  File "/Users/david/server-1/ga4gh/backend.py", line 629, in runSearchRequest
    for obj, nextPageToken in objectGenerator(request):
  File "/Users/david/server-1/ga4gh/backend.py", line 540, in readsGenerator
    intervalIterator = ReadsIntervalIterator(request, readGroup, reference)
  File "/Users/david/server-1/ga4gh/backend.py", line 156, in __init__
    super(ReadsIntervalIterator, self).__init__(request, parentContainer)
  File "/Users/david/server-1/ga4gh/backend.py", line 74, in __init__
    self._pickUpIteration(searchAnchor, objectsToSkip)
  File "/Users/david/server-1/ga4gh/backend.py", line 118, in _pickUpIteration
    assert self._getStart(obj) == searchAnchor
AssertionError

```

**Comment by dcolligan** [comment]
@david4096 good catch.  Fixed in the linked PR

**Comment by dcolligan** [comment]
Fixed in #983
